src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -22,6 +22,8 @@ export const JiraServerUrlPost = async (
 			} else {
 				req.log.debug(`No existing GitHub apps found for url: ${gheServerURL}. Making request to provided url.`);
 				await axios.get(gheServerURL);
+				// TODO: need to install the GithubServerApp and pass that id in the response
+				// This id needs to be passed in the frontend, as it is needed for GH Manual app page
```

Comment:
I don't get this.  The app hasn't been created yet, why would there be an id?

Reply:
When the user adds a URL in the _Jira Server URL_ screen and submits that, that should be saved in the **GitHubServerApps** table, isn't it? 
Once the `GitHubServerApp` has been created, we need its `id` to fetch the other data in the next navigated routes. 
When the user navigates to the _Manual App_ screen, then we need to display the _webhook_, _setup_ and _callback_ URL for the server that was just created. The _webhook_, _setup_ and _callback_ URL need the _githubBaseUrl_ and _uuid_ of the created `GitHubServerApp`. For this reason, we need the `id` of the created `GitHubServerApp`.

Reply:
No, this doesn't happen from that screen. When you submit the url on that screen we do our frontend validation checks then this middleware:

- also verifies the passed value is a valid url
- checks to see if there are any existing GH apps in the table for the provided url
- if there are existing apps, we redirect to the screen you made that lists all the GH apps we found
- if there are no existing apps, we confirm we can hit the provided url (hole in the firewall has been configured) and then redirect to the auto/manual screen.

The auto/manual flow is where we will be saving apps to the GHSA table.

Reply:
This has been reverted